### DISCUSSION TITLE
got error while build packages

### DISCUSSION THREAD

**Discussion by attmore** [termux/termux-packages#17010] [Q&A] [answered]
using ./build-package.sh -I python 

the i got 

cp: cannot create regular file '/home/builder/termux-packages/termux-build/_cache/android-r25b-api-24-v0-tmp/sysroot/usr/include/linux/netfilter/xt_connmark.h': File exists
cp: cannot create regular file '/home/builder/termux-packages/termux-build/_cache/android-r25b-api-24-v0-tmp/sysroot/usr/include/linux/netfilter/xt_dscp.h': File exists
cp: cannot create regular file '/home/builder/termux-packages/termux-build/_cache/android-r25b-api-24-v0-tmp/sysroot/usr/include/linux/netfilter/xt_mark.h': File exists
cp: cannot create regular file '/home/builder/termux-packages/termux-build/_cache/android-r25b-api-24-v0-tmp/sysroot/usr/include/linux/netfilter/xt_rateest.h': File exists
cp: cannot create regular file '/home/builder/termux-packages/termux-build/_cache/android-r25b-api-24-v0-tmp/sysroot/usr/include/linux/netfilter/xt_tcpmss.h': File exists
cp: cannot create regular file '/home/builder/termux-packages/termux-build/_cache/android-r25b-api-24-v0-tmp/sysroot/usr/include/linux/netfilter_ipv4/ipt_ecn.h': File exists
cp: cannot create regular file '/home/builder/termux-packages/termux-build/_cache/android-r25b-api-24-v0-tmp/sysroot/usr/include/linux/netfilter_ipv4/ipt_ttl.h': File exists
cp: cannot create regular file '/home/builder/termux-packages/termux-build/_cache/android-r25b-api-24-v0-tmp/sysroot/usr/include/linux/netfilter_ipv6/ip6t_hl.h': File exists

any help

**Comment by sylirre** [comment]
@attmore The file system expected to be case sensitive. How and where are you running the build environment?

**Comment by attmore** [comment]
![image](https://github.com/termux/termux-packages/assets/1621318/0ab9daf1-43f6-4ca8-8d9c-2794be64b07a)

**Reply by attmore** [reply]
![image](https://github.com/termux/termux-packages/assets/1621318/fdeb02ef-5b47-4dc6-b532-e189bcba3bf3)

**Reply by sylirre** [reply, marked as answer]
Please try in a virtual machine instead. The error is caused by NDK archive which contains a set of files with equal names, for example `xt_DSCP.h` and `xt_dscp.h`. This requires file system to be case sensitive. Otherwise you will see an error that file already exists.

This is not Termux bug. We are neither developers of Android NDK (made by Google) and neither of Linux kernel headers which caused an error.

If Docker on Windows cause issues, use a Linux virtual machine instead.

*P.S. All Termux development is done on Linux. If someone need a compatibility with Windows, then please contribute by sending patches.*

**Comment by attmore** [comment]
thanks for replay 
changed to ubuntu linux env ,it`s ok now
